<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QWidget" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1165</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>100</y>
     <width>101</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>主题</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="textsettings_lb">
      <property name="text">
       <string>文本设置</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>220</y>
     <width>341</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="size_layout" stretch="2,10">
    <item>
     <widget class="QLabel" name="size_lb">
      <property name="text">
       <string>大小</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSlider" name="size_slider">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>270</y>
     <width>341</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="font_layout" stretch="2,10">
    <item>
     <widget class="QLabel" name="font_label">
      <property name="text">
       <string>字体</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="font_cb"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>400</y>
     <width>341</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="buttons_layout">
    <property name="spacing">
     <number>25</number>
    </property>
    <item>
     <widget class="QPushButton" name="save_btn">
      <property name="text">
       <string>保存设置</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancel_btn">
      <property name="text">
       <string>取消设置</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>330</y>
     <width>341</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="color_layout" stretch="1,4,5">
    <property name="spacing">
     <number>30</number>
    </property>
    <item>
     <widget class="QLabel" name="color_lb">
      <property name="text">
       <string>颜色</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="choosecol_btn">
      <property name="text">
       <string>选择颜色</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="empty">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="choosethemebtn">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>140</y>
     <width>131</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>选择预设主题</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>save_btn</sender>
   <signal>clicked()</signal>
   <receiver>Settings</receiver>
   <slot>toPeosonInfo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>412</x>
     <y>427</y>
    </hint>
    <hint type="destinationlabel">
     <x>469</x>
     <y>535</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancel_btn</sender>
   <signal>clicked()</signal>
   <receiver>Settings</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>624</x>
     <y>433</y>
    </hint>
    <hint type="destinationlabel">
     <x>746</x>
     <y>512</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>chooseTheme()</slot>
  <slot>chooseFontColor()</slot>
  <slot>toPeosonInfo()</slot>
  <slot>chooseFontSize()</slot>
  <slot>chooseFontType()</slot>
 </slots>
</ui>
